src/psycop_feature_generation/loaders/raw/load_visits.py

```diff
@@ -91,19 +91,20 @@ def physical_visits(
     }
 
     allowed_visit_types = ("admissions", "ambulatory_visits", "emergency_visits")
-    if visit_type not in allowed_visit_types:
+    if visit_types not in allowed_visit_types:
         raise ValueError(
             f"Invalid visit type. Allowed types of visits are {allowed_visit_types}.",
         )
 
-    if visit_type:
+    if visit_types:
         LPR3_types = {  # pylint: disable=invalid-name
             "admissions": "'Indlæggelse'",
             "ambulatory_visits": "'Ambulant'",
             "emergency_visits": "'Akut ambulant'",
         }
-        d = {key: d[key] for key in ["LPR3", visit_type]}
-        d["LPR3"].where_clause += f" AND pt_type = {LPR3_types[visit_type]}"
+        d = {key: d[key] for key in visit_types + ["LPR3"]}
+        LPR3_types = {key: LPR3_types[key] for key in visit_types}
+        d["LPR3"].where_clause += f" AND pt_type IN ({','.join(LPR3_types.values())})"
```

Comment:
Since you only use the values of the dict, perhaps it should just be a list?

E.g. visit_types_to_include = `[LPR3_types[visit] for visit in visit_types]`.

Reply:
didn't work, because the list then end up in the sql string, which it doesn't like, i.e. "IN (['Indlagt'])" instead og "IN ('Indlagt')"

Reply:
```suggestion
        LPR3_types = [LPR3_types[visit] for visit in visit_types]
        d["LPR3"].where_clause += f" AND pt_type IN ({','.join(LPR3_types)})"
```

You still need the join ;-)